src/Supplier/AddSupplierData/AddSupplierDataName.tsx

```diff
@@ -0,0 +1,30 @@
+import { Container, Grid, Input, Textarea, NumberInput, NumberInputField } from '@chakra-ui/react';
+
+function AddSupplierDataName() {
```

Comment:
Shouldn't it be called just `AddSupplierData`?
`AddSupplierDataName` relates to some name property, but you define a lot of properties down there (phone, comment and so on).

Reply:
I wanted to highlight the differences between the left panel and the right panel